packages/rancher-vsphere/rancher-vsphere-csi/generated-changes/patch/templates/node/daemonset.yaml.patch

```diff
@@ -0,0 +1,23 @@
+--- charts-original/templates/node/daemonset.yaml
++++ charts/templates/node/daemonset.yaml
+@@ -19,7 +19,7 @@
+         role: vsphere-csi
+     spec:
+       {{- if .Values.csiNode.nodeSelector }}
+-      nodeSelector:
++      nodeSelector: {{ include "linux-node-selector" . | nindent 8 }}
+       {{- with .Values.csiNode.nodeSelector }}
+         {{- toYaml . | nindent 8 }}
+       {{- end }}
+@@ -36,9 +36,9 @@
+                 - "windows"
+       {{- end }}
+       {{- if .Values.csiNode.tolerations }}
+-      tolerations:
++      tolerations: {{ include "linux-node-tolerations" . | nindent 8 }}
+       {{- with .Values.csiNode.tolerations }}
+-        {{- toYaml . | nindent 6 }}
++        {{- toYaml . | nindent 8 }}
```

Comment:
nit: indentation was inconsistent, with 6 it looks like:
```
tolerations:
- hi
```

v.s.

```
tolerations:
  - hi
```

Since I'm using 8 around other charts and we use 8 in the csiController anyways, I standardized to 8.